react/features/overlay/components/AbstractReloadableOverlay.js

```diff
@@ -0,0 +1,20 @@
+/* global APP */
+import AbstractOverlay from './AbstractOverlay';
+
+/**
+ * Implements an abstract React Component for overlay with ability to
+ * reload the page.
+ */
+export default class AbstractReloadableOverlay extends AbstractOverlay {
```

Comment:
The only reason for this was the rule in our code style guide "No duplicated code".